### PR TITLE
Percentage youth

### PR DESCRIPTION
**Pull request by xybrnet**
@longhotsummer

### CODE REVIEW

File: sql/youth_population.sql [hunk 1]
```diff
@@ -25,16 +25,16 @@ SET default_with_oids = false;
 CREATE TABLE youth_population (
     geo_level character varying(16) NOT NULL,
     geo_code character varying(8) NOT NULL,
-    total_pop integer,
-    youth_pop integer
+    youth_pop integer,
```
Comment (longhotsummer) [inline sql/youth_population.sql]: 🙈

File: wazimap_za/profiles/youth.py [hunk 2]
```diff
@@ -43,10 +43,16 @@ def get_profile(geo_code, geo_level, profile_name=None):
 
 
 def get_demographics_profile(geo_code, geo_level, session):
-    pop_dist_data, pop_total = get_stat_data(
-            ['population group'], geo_level, geo_code, session)
+    table = get_datatable('youth_population').model
```
Comment (longhotsummer) [inline wazimap_za/profiles/youth.py]: You should also be able to use `SimpleTable.get_stat_data` after getting the table object. This saves you from having to care about the db at all.

**Comment by longhotsummer** [comment]
Looks good. Don't forget to update the data when you push.

File: sql/youth_population.sql (hunk 1 above)
Comment (xybrnet) [inline sql/youth_population.sql]: I know...

File: wazimap_za/profiles/youth.py (hunk 2 above)
Comment (xybrnet) [inline wazimap_za/profiles/youth.py]: I was thinking this exists for a SImpleTable. Thanks - I've updated.

**Pull request merged**